Floofbot/Modules/Utilities.cs

```diff
@@ -256,4 +258,163 @@ public async Task ServerList()
+
+        [Command("convert")]
+        [Alias("conv")]
+        [Summary("Converts units to other units, such as Celcius to Fahrenheit.")]
+
+        public async Task convert(string input)
+        {
+            Regex fahReg = new Regex("\\b(\\d+)(f)\\b", RegexOptions.IgnoreCase);
+            Regex celReg = new Regex("\\b(\\d+)(c)\\b", RegexOptions.IgnoreCase);
+            Regex miReg = new Regex("\\b(\\d+)(mi)\\b", RegexOptions.IgnoreCase);
+            Regex kmReg = new Regex("\\b(\\d+)(km)\\b", RegexOptions.IgnoreCase);
+            Regex kgReg = new Regex("\\b(\\d+)(kg)\\b", RegexOptions.IgnoreCase);
+            Regex lbReg = new Regex("\\b(\\d+)(lbs)\\b", RegexOptions.IgnoreCase);
+
+            if (fahReg.Match(input).Success)
+            {
+                Match m = fahReg.Match(input);
+
+                Group g = m.Groups[1];
+
+                string FahStr = Convert.ToString(g);
+                double FahTmp = Convert.ToDouble(FahStr);
+
+                Temperature Fah = Temperature.FromDegreesFahrenheit(FahTmp);
+                double Cel = Fah.DegreesCelsius;
+
+                EmbedBuilder builder = new EmbedBuilder()
+                {
+                    Title = "Temperature conversion",
+                    Description = $"🌡 {(Temperature)Fah} is equal to {(double)Math.Round(Cel, 2, MidpointRounding.ToEven)}°C.",
+                    Color = EMBED_COLOR
+                };
+
+                await Context.Channel.SendMessageAsync("", false, builder.Build());
+            }
+            else if (celReg.Match(input).Success)
+            {
+                Match m = celReg.Match(input);
+
+                Group g = m.Groups[1];
+
+                string celStr = Convert.ToString(g);
+                double celTmp = Convert.ToDouble(celStr);
+
+                Temperature Cel = Temperature.FromDegreesCelsius(celTmp);
+                double Fah = Cel.DegreesFahrenheit;
+
+                EmbedBuilder builder = new EmbedBuilder()
+                {
+                    Title = "Temperature conversion",
+                    Description = $"🌡 {(Temperature)Cel} is equal to {(double)Math.Round(Fah, 2, MidpointRounding.ToEven)}°F.",
+                    Color = EMBED_COLOR
+                };
+
+                await Context.Channel.SendMessageAsync("", false, builder.Build());
+            }
+            else if (miReg.Match(input).Success)
+            {
+                Match m = miReg.Match(input);
+
+                Group g = m.Groups[1];
+
+                string miStr = Convert.ToString(g);
+                double miTmp = Convert.ToDouble(miStr);
+
+                Length mi = Length.FromMiles(miTmp);
+                double km = mi.Kilometers;
+
+                EmbedBuilder builder = new EmbedBuilder()
+                {
+                    Title = "Length conversion",
+                    Description = $"📏 {(Length)mi} is equal to {(double)Math.Round(km, 3, MidpointRounding.ToEven)}Km.",
```

Comment:
Actually, looking at it now, I would just move the rounding to the variable declaration

`double km = Math.Round(mi.Kilometers, 3, MidpointRounding.ToEven)`